parser-utils/src/query.rs

```diff
@@ -11,7 +11,6 @@
 extern crate combine;
```

Comment:
These utilities aren't really query specific, they're `edn::Value` specific.  (Am I wrong?)  Can we move to `src/value_parsing.rs` or something you prefer?